Comanche055/JET_SELECTION_LOGIC.agc

```diff
@@ -278,7 +278,7 @@ BDZCHECK	CA	ZNDX
 		BZF	TAUCHECK
 		
 ACRBDZ		CA	T5TEMP		# Z-TRANSLATION ACCEPTED EVEN THO WE MAY
-		TS	RWORD1		# HAVE INTRODUCED AN UNDESIREABLE ROLL
```

Comment:
wopian at #91: Typo present in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1047.jpg).